node/rpc-client/src/lib.rs

```diff
@@ -160,18 +160,18 @@ where
     let mut http_res = match api_info.token.to_owned() {
         Some(jwt) => client
             .post(api_url)
-            .content_type("application/json-rpc")
+            .content_type("application/json")
             .body(surf::Body::from_json(&rpc_req)?)
             .header("Authorization", jwt),
         None => {
             if let Some(jwt) = token {
                 surf::post(api_url)
-                    .content_type("application/json-rpc")
+                    .content_type("application/json")
```

Comment:
It seems that content type is not specified in the [spec](https://www.jsonrpc.org/specification#request_object), `axum::Json` extractor requires `application/json`, or we need to parse json from string